extensions/vertx-http/deployment/src/main/resources/dev-templates/tags/actionableExtension.html

```diff
@@ -17,8 +17,10 @@
                 {#if it.metadata.status?? == 'experimental'}
                 <span class="float-right badge badge-warning">EXPERIMENTAL</span>
                 {/if}
+                {#if it.description??}
                 <span class="card-subtitle mb-2 text-muted">{it.description}</span>
                 <br/>
+                {/if}
```

Comment:
I also thought of showing that no description exists explicitly: 

```suggestion
                <span class="card-subtitle mb-2 text-muted">{it.description ?: 'No description'}</span>
                <br/>
```

Any thoughts?

Reply:
![image](https://user-images.githubusercontent.com/54133/130272255-22f359dc-ec73-4db0-9740-8bc89ccd47d3.png)